packages/eslint/package.json

```diff
@@ -13,5 +13,10 @@
     "eslint": "^8.43.0",
     "eslint-config-prettier": "^8.8.0",
     "eslint-plugin-import": "^2.27.5"
+  },
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "8.3",
+    "@typescript-eslint/parser": "8.3",
```

Comment:
I don't believe we actually need these as dev dependencies? Are you seeing any issues with them? If there was/is an issue with the versions than `npm` would tell us anyways.

Reply:
Oh, my bad, I didn't notice that they were peer dependencies.